fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -399,15 +399,17 @@ struct TenantManagementWorkload : TestWorkload {
 		std::string prefix;
 		std::string base64Prefix;
 		std::string printablePrefix;
+		bool encrypted;
```

Comment:
This test should set the encrypted value and make sure that you get the correct value for the encrypted field for each tenant.

Reply:
+1